# fix gcc-3.3.6 "bool inline" vs "inline bool" warning, which might be considered
#     as an error with other compiler versions

--- a/rosetta++/packing_measures.h	2008-04-17 22:27:00.000000000 +0200
+++ b/rosetta++/packing_measures.h	2008-12-12 12:30:30.000000000 +0100
@@ -376,13 +376,13 @@
 
 
 		// methods for checking params
-		bool inline cavity_ball_size_ok(int ii) const {
+		inline bool cavity_ball_size_ok(int ii) const {
 			return packing_options::min_hole_radius <= cavity_balls_[ii].radius() && cavity_balls_[ii].radius() <= packing_options::max_hole_radius;
 		}
-		bool inline cavity_ball_buried(int const ii) const {
+		inline bool cavity_ball_buried(int const ii) const {
 			return cavity_balls_[ii].sasa() <= packing_options::max_hole_sasa;
 		}
-		bool inline cluster_ok( CavityBallCluster const & cbc ) const {
+		inline bool cluster_ok( CavityBallCluster const & cbc ) const {
 			//cout << cbc.str() << endl;
 			//cout <<  packing_options::min_cluster_ball_radius << ' ' << packing_options::min_cluster_size << ' ' << packing_options::min_cluster_size_aspiration << endl;
 			return packing_options::min_cluster_volume       <= cbc.volume_ &&
@@ -391,16 +391,16 @@
 			       packing_options::max_cluster_ball_radius  >= cbc.largest_cavity_ball_radius()
 			;
 		}
-		bool inline cluster_ok( CavityBallCluster const * const cbc ) const {
+		inline bool cluster_ok( CavityBallCluster const * const cbc ) const {
 			return cluster_ok(*cbc);
 		}
-		bool inline cavity_ball_cluster_ok(int const ii) const {
+		inline bool cavity_ball_cluster_ok(int const ii) const {
 			if (cavity_balls_[ii].cluster_id_ < 1)
 				return false;
 			else
 				return cluster_ok( cavity_balls_[ii].cluster() );
 		}
-		bool inline cavity_ball_all_ok(int const ii,
+		inline bool cavity_ball_all_ok(int const ii,
 																				 bool const require_size_ok    = true,
 																				 bool const require_buried     = true,
 																				 bool const require_cluster_ok = true  ) const
